src/http/client.cr

```diff
@@ -117,12 +117,12 @@ class HTTP::Client
   {% else %}
     def initialize(@host : String, port = nil, tls : Bool | OpenSSL::SSL::Context::Client = false)
       @tls = case tls
-             when true
-               OpenSSL::SSL::Context::Client.new
+             when Bool
```

Comment:
No, it's just my mistake. Firstly I implemented exhaustiveness check without supporting `true` and `false`. It is remains of this.